configure

```diff
@@ -1033,6 +1033,7 @@ def glob_to_var(dir_base, dir_sub, patch_dir):
   return list
 
 def configure_intl(o):
+  config_vars = o['variables']
```

Comment:
Sorry about that, it was the "old me", that tried to be cheeky.
Anyway I reverted, as requested.